packages/ibm-products/src/components/BigNumbers/BigNumbers.tsx

```diff
@@ -1,92 +1,89 @@
+  Characters,
+  DefaultLocale,
   formatValue,
   getIconSize,
-  getSupportedLocale,
-  BigNumbersSize,
-  DefaultLocale,
-  Characters,
 } from './constants';
 
-// The block part of our conventional BEM class names (blockClass__E--M).
 const blockClass = `${pkg.prefix}--big-numbers`;
-const skeletonBlockClass = `${pkg.prefix}--big-numbers-skeleton`;
 const componentName = 'BigNumbers';
 
 // NOTE: the component SCSS is not imported here: it is rolled up separately.
 
-// Default values for props
-const defaults = {
-  forceShowTotal: false,
-  fractionDigits: 1,
-  loading: false,
-  locale: DefaultLocale,
-  percentage: false,
-  size: BigNumbersSize.Default,
-  total: 0,
-  trending: false,
-  truncate: true,
-  value: null,
-};
+export interface BigNumbersProps {
+  className?: string;
+  forceShowTotal?: boolean;
+  fractionDigits?: number;
+  iconButton?: ReactNode;
+  loading?: boolean;
+  label: string;
+  locale?: string;
+  percentage?: boolean;
+  size?: 'default' | 'lg' | 'xl';
+  tooltipDescription?: string;
+  total?: number;
+  trending?: boolean;
+  truncate?: boolean;
+  value?: number;
+}
 
 /**
  * BigNumbers is used to display large values in a small area. The display of
- * values can be the value itself, or grouped in a numerator / denominator fashion.
+ * values can be the value itself, or grouped in a `numerator/denominator` fashion.
  * Control over the total fraction decimals displayed as well as how the
  * values/totals are displayed are done via a locale prop. Other optional props
  * allow control over size, truncation, if the value is a percentage, the addition
  * of a button as well as tool tip functionality.
  * The default locale is English (`en-US`) if one is not provided or if the provided one is not supported.
  */
-export let BigNumbers = React.forwardRef(
+export let BigNumbers = forwardRef<HTMLDivElement, BigNumbersProps>(
   (
     {
       // The component props, in alphabetical order (for consistency).
       className,
-      forceShowTotal = defaults.forceShowTotal,
-      fractionDigits = defaults.fractionDigits,
+      forceShowTotal = false,
+      fractionDigits = 1,
       iconButton,
-      loading = defaults.loading,
+      loading = false,
       label,
-      locale = defaults.locale,
-      percentage = defaults.percentage,
-      size = defaults.size,
+      locale = DefaultLocale,
+      percentage = false,
+      size = 'default',
       tooltipDescription,
-      total = defaults.total,
-      trending = defaults.trending,
-      truncate = defaults.truncate,
-      value = defaults.value,
+      total,
+      trending = false,
+      truncate = true,
+      value,
       // Collect any other property values passed in.
       ...rest
     },
     ref
   ) => {
-    const BigNumbersClasses = classnames(className, {
-      [`${blockClass}--lg`]: size === BigNumbersSize.Large,
-      [`${blockClass}--xl`]: size === BigNumbersSize.XLarge,
+    const BigNumbersClasses = cx(className, {
```

Comment:
another small stylistic observation: i think as a general react convention that uppercase variables like this are reserved for components. could you just set this to lowercase?

Reply:
Done